pkg/fission-cli/cmd/function/getmeta.go

```diff
@@ -46,10 +44,16 @@ func (opts *GetMetaSubCommand) do(input cli.Input) error {
 		return errors.Wrap(err, "error getting function")
 	}
 
-	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
-	fmt.Fprintf(w, "%v\t%v\n", "NAME", "ENV")
-	fmt.Fprintf(w, "%v\t%v\n", fn.ObjectMeta.Name, fn.Spec.Environment.Name)
-	w.Flush()
+	fmt.Printf("Name: %v\n", fn.ObjectMeta.Name)
+	fmt.Printf("Environment: %v\n", fn.Spec.Environment.Name)
+	fmt.Println("Labels:")
+	for k, v := range fn.ObjectMeta.Labels {
+		fmt.Printf("  %s=%s\n", k, v)
+	}
+	fmt.Println("Annotations:")
```

Comment:
Should we avoid printing this string if annotations is empty ?

Reply:
Is it ok list this? I am new to both go and PR process so excuse me if any mistakes.